plugins/org.jkiss.dbeaver.model.jdbc/src/org/jkiss/dbeaver/model/impl/jdbc/JDBCDataSource.java

```diff
@@ -646,6 +646,11 @@ private static int getValueTypeByTypeName(@NotNull String typeName, int valueTyp
             if (SQLConstants.DATA_TYPE_TINYINT.equalsIgnoreCase(typeName)) {
                 return Types.TINYINT;
             }
+        } else if (valueType == Types.ROWID) {
```

Comment:
I don't think it should be here. If you think that other drivers may benefit from it, then please remove database-specific comment.